packages/jest-circus/src/index.ts

```diff
@@ -63,7 +64,33 @@ const _dispatchDescribe = (
     mode,
     name: 'start_describe_definition',
   });
-  blockFn();
+  const describeReturn = blockFn();
+
+  // TODO throw in Jest 25
+  if (isPromise(describeReturn)) {
+    console.warn(
+      formatExecError(
+        new Error(
```

Comment:
instead of a new error, you could reuse the stack from `asyncError` above.


`formatExecError({message: 'current message', stack: asyncError.stack})`. Should remove `_dispatchDescribe` from the stack.

If you don't wanna reuse it, do `formatExecError(new ErrorWithStack('current message', describeFn))` instead of `new Error`